src/containers/home/components/7-climate-crisis.tsx

```diff
@@ -3,59 +3,49 @@ import SectionTitle from 'components/section-title';
 
 const ClimateCrisis = () => {
   return (
-    <div className="min-h-[150vh] sm:min-h-screen">
-      <div className="absolute left-0 h-full min-h-[150vh] w-full overflow-hidden bg-white sm:min-h-screen">
-        <div className="container flex h-screen min-h-screen flex-col justify-between gap-6 pt-14 lg:mt-0 lg:justify-start lg:gap-0">
-          <div className="flex-0">
-            <SectionTitle className="mb-4" color="green">
-              Climate crisis and inequality.
-            </SectionTitle>
-            <SectionSubtitle size="large" className="text-900">
-              The climate crisis multiplies the threats of existing inequality.
-            </SectionSubtitle>
+    <div className="min-h-fit w-full overflow-hidden bg-white">
+      <div className="flex min-h-screen flex-col justify-between gap-6 pt-14 lg:mt-0 lg:h-screen lg:justify-start lg:gap-0 lg:pt-[104px]">
```

Comment:
I think a h-full will help to fix the problem that sometimes the image and text are cut:
<img width="1440" alt="image" src="https://github.com/Vizzuality/climate-inequality/assets/9701591/8ecfee5f-bde5-4eda-b60b-582543e4b763">